README.md

```
@@ -1,61 +1,153 @@
+  ![](https://github.com/user-attachments/assets/852bc660-8539-4c64-a2b7-7c03108e9ec4)
+
+- 체험 수정/삭제
+  
+  ![](https://github.com/user-attachments/assets/036cb1df-fef4-4dff-a2a7-531122666271)
+
+<br/>
+
+
+### 체험 예약
+
+![](https://github.com/user-attachments/assets/7fac5e71-a918-4bf0-9e35-91416dda36c6)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)